fake-snippets-api/routes/api/orders/create.ts

```diff
@@ -198,11 +197,6 @@ export default withRouteSpec({
     newOrder.has_error = true
   }
 
-  if (newOrder.has_error) {
-    return ctx.json({
-      order: newOrder,
-    })
-  }
   const order = ctx.db.addOrder(newOrder)
```

Comment:
Also /orders/create should never return an order with an error, the error is always found on the mext poll, because this is the behavior of the real api

Reply:
If for some reason we cant create the order synchronously (eg bad request params) we return { error } and dont create an order. But this doesnt meed the simulate scenario